modules/cloud-sql/variables.tf

```diff
@@ -247,12 +247,7 @@ variable "resource_timeout" {
 
 variable "additional_users" {
   description = "Additional database users"
-  type = list(object({
-    name = string
-    # TODO: once optional object parameters are supported
-    # password = string
-    # length = number
-  }))
+  type = list
```

Comment:
@ciiqr If new additional username is added (along with the existing users), the elements within the list are first ordered internally, and then it (may) forces replacement of existing users in the list in order to create a new one.
As a result, `existing_user` would be removed and `new_user` will be created. And a second instance of `existing_user` is not being re-created by TF (!).

Reply:
This is fine for now. I know tf was planning to fix these sort of issues at some point, but this seems reasonable enough for now, we can always make more changes if it ever becomes relevant.